### PR TITLE
Tag relative option with no argument

### LINKED ISSUE #2096: Undocumented incompatible change to --tag-relative
Original ctags accepts --tag-relative without an option which universal ctags does not.
It leads to incompatibility that is not documented and IMO is not necessary.
I am dealing with an old tool relaying on ctags and prefer this option to be compatible to original verion.

The command line you used to run ctags:

```
$ ctags --options=NONE --tag-relative -R -f - .
ctags: Notice: No options will be read from files or environment
ctags: Invalid value for "tag-relative" option```
```

The version of ctags:

```
$ ctags --version
Universal Ctags 0.0.0(c8405be), Copyright (C) 2015 Universal Ctags Team
Universal Ctags is derived from Exuberant Ctags.
Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
  Compiled: May 14 2019, 17:34:10
  URL: https://ctags.io/
  Optional compiled features: +wildcards, +regex, +iconv, +option-directory, +xpath, +case-insensitive-filenames, +packcc
```

### PR DESCRIPTION
**Pull request by masatake**
Close #2096.

### CODE REVIEW

**Comment by masatake** [comment]
@bogdan, if you are good at English, can I ask you to review the update of the man page in this pull requeust?

File: man/ctags.1.rst.in [hunk 1]
```diff
@@ -1420,12 +1420,16 @@ are not listed here. They are experimental or for debugging purpose.
 	name. [Ignored in etags mode]
 
 ``--tag-relative[=yes|no]``
```
Comment (bogdan) [inline man/ctags.1.rst.in]: You don't have `always` and `never` mentioned in options list

File: man/ctags.1.rst.in [hunk 2]
```diff
@@ -1420,12 +1420,16 @@ are not listed here. They are experimental or for debugging purpose.
...
-	Indicates that the file paths recorded in the tag file should be
+	Specyfing yes indicates that the file paths recorded in the tag file should be
```
Comment (bogdan) [inline man/ctags.1.rst.in]: I would follow the format language in other options if you have a doubt (see --sort for example):

```
The yes value  (the default when unspecified) indicates...
The X value indicates...
etc
```

**Comment by masatake** [comment]
Thank you. Updated.
`(the default when unspecified)` is not used because it is written in a later sentence.

**Comment by masatake** [comment]
Thank you!

**Pull request merged**